awx/main/models/inventory.py

```diff
@@ -943,7 +926,6 @@ class Meta:
 
     source = models.CharField(
         max_length=32,
-        choices=SOURCE_CHOICES,
```

Comment:
@chrismeyersfsu any ideas if this should be kept?

Reply:
We confirmed we do not need this one and I will be pushing up a corresponding migration shortly!